docs/HyperIndex/benchmarks.md

```diff
@@ -0,0 +1,41 @@
+---
+id: benchmarks
+title: HyperIndex Benchmarks
+sidebar_label: Benchmarks
+slug: /benchmarks
+---
+
+# HyperIndex Performance Benchmarks
+
+## Overview
+
+HyperIndex delivers industry-leading performance for blockchain data indexing. Independent benchmarks have consistently shown Envio's HyperIndex to be the fastest indexing solution available, with dramatic performance advantages over competitive offerings.
+
+## Recent Independent Benchmarks
+
+The most comprehensive and up-to-date benchmarks were conducted by Sentio in April 2025 and are available in the [sentioxyz/indexer-benchmark repository](https://github.com/sentioxyz/indexer-benchmark). These benchmarks compare Envio's HyperIndex against other popular indexers across multiple real-world scenarios:
+
+### Key Performance Highlights
+
+| Case                    | Description              | Envio  | Nearest Competitor | Advantage   |
+| ----------------------- | ------------------------ | ------ | ------------------ | ----------- |
+| LBTC Token Transfers    | Basic event indexing     | 2m     | 8m (Sentio)        | 4x faster   |
+| Block Processing        | 100K Ethereum blocks     | 7.9s   | 10m (Subgraph)     | 75x faster  |
+| Transaction Analysis    | Gas usage analytics      | 1m 26s | 5m (Subsquid)      | 3.5x faster |
+| Trace Processing        | Uniswap V2 swap analysis | 41s    | 2m (Subsquid)      | 3x faster   |
+| Template-based Indexing | Uniswap V2 template      | 20s    | 2m (Subsquid)      | 6x faster   |
```

Comment:
Really nice. I think it'd be cool to also include a column for `Ponder` and `TheGraph` since they are the main names in indexing currently. And there is much bigger advantage compared to them

Reply:
case_2_lbtc_full is also very interesting.